src/medUtilities/medUtilitiesVTK.h

```diff
@@ -86,4 +86,7 @@ class MEDUTILITIES_EXPORT medUtilitiesVTK
                            double* mean,
                            double* stdDev,
                            int component = 0);
+
```

Comment:
Indentation, replace 'indice' with 'index', add documentation:

Extracts a single mesh from a 4D sequence of meshes.
data -> 4D sequence
index -> index of the mesh to extract in the sequence